tempCodeRunnerFile.javascript

```diff
@@ -0,0 +1,3 @@
+const reg = new RegExp('test');
+
+console.log(reg.test('asdklaskdas;ldtestasdasd'));
```

Comment:
lol, what's this?

Reply:
heh whoops, I have a vs code extension I use to run code snippets, apparently it created a file